addons/web/static/src/js/views/pivot/pivot_model.js

```diff
@@ -322,21 +322,46 @@ var PivotModel = AbstractModel.extend({
      * @param {'row'|'col'} type
```

Comment:
The commit message is not very clear (mainly the last part). Could you rewrite it?